app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -527,6 +527,9 @@ private void handleNavIntent() {
             long feedId = intent.getLongExtra(EXTRA_FEED_ID, 0);
             if (tag != null) {
                 loadFragment(tag, args);
+                if (intent.hasExtra(MainActivityStarter.EXTRA_OPEN_DRAWER) && drawerLayout != null) {
+                    drawerLayout.open();
+                }
```

Comment:
Now this extra only works when passing a fragment tag at the same time. What if I want to open a podcast and open the drawer at the same time? Or just start the app on the default fragment, but with the drawer open? I would just move this drawer check outside the code that handles opening a specific page.